test/uk/gov/hmrc/helptosave/utils/TestSupport.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.utils
+
+import org.joda.time.LocalDate
+import org.scalamock.scalatest.MockFactory
+import uk.gov.hmrc.helptosave.config.WSHttp
+import uk.gov.hmrc.play.http.HeaderCarrier
+
+trait TestSupport extends MockFactory {
+
+  implicit val hc = HeaderCarrier()
+  val mockHttp = mock[WSHttp]
+
+  val nino = "nino"
+
+  val date = new LocalDate(2017, 6, 12) // scalastyle:ignore magic.number
```

Comment:
We probably don't use `joda.time.LocalDate` enough to warrant putting this into the common test trait - if you take it out then `javaDate` can just become `date`